starter-code/src/Main.jsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import Nav from './nav'
+
+export default function Main() {
+    return (
+        <div className="Main">
+            <Nav logoIronhack="./../images/ironhack-logo.svg" menuLogo="./../images/menu-top.svg"></Nav>
```

Comment:
You are not supposed to access anything outside src.

If you want to reference assets defined in the public folder you just have to link them as follows: 

` <Nav logoIronhack="/images/ironhack-logo.svg" menuLogo="/images/menu-top.svg"></Nav>`

Eventhough it was working, it breaks sometimes and it might break when deployed/built